region/crateria/central.json

```diff
@@ -2224,13 +2442,134 @@
         {
           "from": 4,
           "to": [
+            {
+              "id": 2,
+              "strats": [
+                {
+                  "name": "G-Mode Morph through Bomb Blocks",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [4],
+                      "mode": "any",
+                      "artificialMorph": true
+                    }},
+		    {"or": [
+		      "SpringBall",
+		      "Bombs",
+		      {"and": [
+		        "Morph",
+		        "can3HighMidAirMorph"
```

Comment:
There should be a helper made for `h_canThreeTileJumpMorph` that would include `h_canTwoTileJumpMorph`